src/ddprof.cc

```diff
@@ -42,16 +42,16 @@ void sigsegv_handler(int sig, siginfo_t *si, void *uc) {
   static void *buf[k_stacktrace_buffer_size] = {};
   size_t const sz = backtrace(buf, std::size(buf));
 #endif
-  (void)fprintf(
-      stderr, "ddprof[%d]: <%.*s> has encountered an error and will exit\n",
-      getpid(), static_cast<int>(str_version().size()), str_version().data());
+  const char msg[] = "ddprof: encountered an error and will exit\n";
```

Comment:
Thanks for raising this. We have more work to improve crash tracking, though this would be a reasonable first step. 
You are removing a minor info here (the si->si_addr), I'll add this back in while keeping your changes.
The GLIBC version can stay as is as we do not deploy that version (we use a musl build for the releases).